packages/react-native/Libraries/NewAppScreen/components/NewArchitectureBadge.js

```diff
@@ -17,11 +17,13 @@ import useColorScheme from '../../Utilities/useColorScheme';
 import Colors from './Colors';
 import React from 'react';
 
-const HermesBadge = (): Node => {
+const NewArchitectureBadge = (): Node => {
   const isDarkMode = useColorScheme() === 'dark';
-  const version =
+  const hermesVersion =
     global.HermesInternal?.getRuntimeProperties?.()['OSS Release Version'] ??
     '';
+  const renderer = global?.nativeFabricUIManager ? 'Fabric, New Architecture' : 'Paper';
```

Comment:
New Architecture includes Fabric (and others), so let's just say "new architecture".

The presence of `nativeFabricUIManager` itself is no guarantee this component is rendered using it though.